superbench/benchmarks/micro_benchmarks/cublas_test.py

```diff
@@ -0,0 +1,307 @@
+                    print(config_json_str)
+                    complete_command = command + (' --config_json ') + config_json_str
+                    self._commands.append(complete_command)
+
+            else:
+                custom_config_str = yaml.safe_load(self._args.config_json_str)
+                config_json_str = "\'" + json.dumps(custom_config_str).replace(' ', '') + "\'"
+                complete_command = command + (' --config_json ') + config_json_str
+                self._commands.append(complete_command)
+        except BaseException as e:
+            logger.error('Invalid input params - benchmark: {},  message: {}'.format(self._name, str(e)))
+            return False
+        return True
+
+    def _process_raw_result(self, cmd_idx, raw_output):
+        """Function to process raw results and save the summarized results.
+
+          self._result.add_raw_data() and self._result.add_result() need to be called to save the results.
+
+        Args:
+            cmd_idx (int): the index of command corresponding with the raw_output.
+            raw_output (str): raw output string of the micro-benchmark.
+
+        Return:
+            True if the raw output string is valid and result can be extracted.
+        """
+        self._result.add_raw_data('raw_output_' + str(cmd_idx), raw_output)
+
+        try:
+            lines = raw_output.split('\n')
+            metric = ''
+            error = False
+            raw_data = []
+            for line in lines:
+                if '[function config]' in line:
+                    metric = line[line.index('[function config]: ') + len('[function config]: '):]
+                if '[raw_data]' in line:
+                    raw_data = line[line.index('[raw_data]: ') + len('[raw_data]: '):]
+                    raw_data = raw_data.split(',')
+                    raw_data.pop()
+                    raw_data = [float(item) for item in raw_data]
+                    self._result.add_result(metric, sum(raw_data) / len(raw_data))
+                    self._result.add_raw_data(metric, raw_data)
+                if 'Error' in line:
+                    error = True
+        except BaseException as e:
+            logger.error(
+                'Cannot extract results from cublas functions - round: {}, index of cmd: {}, \
+                benchmark: {}, raw data: {}, message: {}'.format(
+                    self._curr_run_index, cmd_idx, self._name, raw_output, str(e)
+                )
+            )
+            return False
+        if error:
+            logger.error(
+                'Error in running cublas test - round: {}, index of cmd: {}, benchmark: {}, raw data: {}'.format(
+                    self._curr_run_index, cmd_idx, self._name, raw_output
+                )
+            )
+            return False
+        return True
+
+
+BenchmarkRegistry.register_benchmark('cublas-test', CublasBenchmark)
```

Comment:
Specify the platform, refer:
`BenchmarkRegistry.register_benchmark('cublas-test', CublasBenchmark, platform=Platform.CUDA)`

Reply:
ok, thank you for the comment